frontend/src/components/module-editor/EditContentOptionsMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { HStack, IconButton } from "@chakra-ui/react";
+import { EditIcon, CopyIcon, DeleteIcon } from "@chakra-ui/icons";
+import { EditContentOptionsMenuProps } from "../../types/ModuleEditorTypes";
+
+const EditContentOptionsMenu = ({
+  isVisible,
+  onEditClick,
+  onCopyClick,
+  onDeleteClick,
+}: EditContentOptionsMenuProps): React.ReactElement => {
+  const currentTabIndex = isVisible ? 0 : -1;
+
+  return (
+    <HStack spacing={4} opacity={isVisible ? 1 : 0}>
+      <IconButton
+        aria-label="Edit this content block"
+        tabIndex={currentTabIndex}
```

Comment:
If we do the conditional rendering, we might also be able to avoid having to set the `tabIndex` manually.